R/nretention.r

```diff
@@ -43,7 +43,7 @@ calc_nleach <- function(B_BT_AK, B_LU_BRP, B_GT, D_NLV, B_LG_CBS, leaching_to){
                                                  'Zuidwestelijk Akkerbouwgebied','Rivierengebied','Hollands/Utrechts Weidegebied',
                                                  'Waterland en Droogmakerijen','Westelijk Holland','IJsselmeerpolders',
                                                  'Centraal Veehouderijgebied','Oostelijk Veehouderijgebied','Noordelijk Weidegebied',
-                                                 'Veenkolonien en Oldambt','Bouwhoek en Hogeland'), empty.ok = FALSE)
+                                                 'Veenkoloni\u00EBn en Oldambt','Veenkolonien en Oldambt','Bouwhoek en Hogeland'), empty.ok = FALSE)

```

Comment:
In f4150921f204251d07261ee1d76c93613117ed0e yuki removed the umlaut, so why would we know allow it?

Reply:
In bodemschat B_LG_CBS contains "Veenkoloniën en Oldambt", for fewer preprocessing steps on the users part, this should be accepted.
OBIC version 0.11.1 does not accept "Veenkoloniën en Oldambt"

Reply:
Ok, it also looks like these functions are the only one using `B_LG_CBS`